### ISSUE TITLE
Possible changes to Dumping Ground Portrayal and A & I

### ISSUE THREAD

**Issue by alvarosanuy** [S-101-Portrayal-subWG/Working-Documents#44]
To be discussed by the Port subWG

- [ ] Changes to portrayal
- [ ] Changes to A & I

**Below is feedback from user as submitted to the UKHO:**

Good Day, 

Please note following ECDIS limitations noted, posing a danger to safe navigation. 

ECDIS Limitation:

It is noted that, Dumping Ground on ECDIS is marked by magenta boundary and a normal [ i ].  A route passing over a Dumping Ground does not give an audio or visual alert to the navigator.

Dumping Ground, by the nature of the work for which it is used, it poses a risk of reduced depth. Therefore, it is recommended that, it may be categories as a cautionary area and have an appropriate symbol.

Any route passing over a Dumping Ground shall give an audio and visual alert to the navigator. It shall also alert the navigator during the passage plan and route check stage. 

On paper chart Dumping Ground were clearly marked, however with the present arrangement of ECDIS, even with ALL features selected on chart presentation, a Dumping Ground is not conspicuously displayed. A “Dumping Ground” note only appears, once the area in question is interrogated. 

![image](https://user-images.githubusercontent.com/70189016/150028505-b5274c6b-1df7-478e-82cd-608a732792db.png)

Dumping Ground shall be treated as cautionary area and ECDIS shall give visual and audio warning when route passes over this area.
 

Vessel’s ECDIS details:

Make – Tokyo Keiki

Model – EC-8600

Attachments:

Dumping Ground – Comparisons between Paper chart and ECDIS presentation
Route Check comparison – Over Dumping ground and across anchorage
H102 – Hydrographic note
[Attachements.zip](https://github.com/S-101-Portrayal-subWG/Working-Documents/files/7892796/Attachements.zip)

**Comment by alvarosanuy** [comment]
I think that this request is triggered by the lack of frequent chart updates to reflect the real depths in the area. As an example, in Australia ports undertake a post dredge survey of dumping areas and the data is submitted to the HO. We usually issue an ENC update (CTNARE over dredged channels and dumping grounds) alerting mariners dredging operations are underway. Later on, we use the survey of the dumping ground to update the ENC content (usually by ENC update). We are confident that the charted least depth + CATZOC accuracies convey enough info for the mariner to trust the charted information. In the example supplied by the UKHO it seems that the dumping grounds have not been surveyed for a very long time. Maybe the UOC/DCEG should recommend the 'double encoding' of CTNAREs and/or the downgrading of the charted CATZOC when the HO can't guarantee that its accuracies cover the differences between charted and real depths. From our point of view, having another feature triggering A&I when we are actively managing the chart content does not seem the best way to go.

**Comment by alvarosanuy** [comment]
**Decision made at Portrayal subWG meeting on 14/7/22**

1. Pass this Issue to the DCEG SubWG to review existing encoding guidance regarding best practices for the encoding of **DumpingGround** when post-dredged surveys are not submitted to HOs. 
2. Leave this issue open for member states to comment on:
- [ ] The benefit of including **DumpingGround** **categoryOfDumpingGround**=5 (spoil ground) in the A&I catalogue (Indication - Yellow highlight). 
- [ ] The benefit of allocating **DumpingGround** **categoryOfDumpingGround**=5 (spoil ground) its own symbology in CHGRD colour instead of Magenta. An option could be using the existing  _Symbolised_ and _Plain_ (Point & Surface) symbology as per S-52 (DMPGRD) but in CHGRD colour instead (???) RESTRN01 would still be run and, if required, relevant magenta symbols would be added.

![image](https://user-images.githubusercontent.com/70189016/211972080-907f68a8-542b-4a21-8c45-3252f41c7521.png)

**Comment by alvarosanuy** [comment]
**Portrayal subWG meeting - 12th January 2023**

1. No decision made. The issue will continue open and the **UKHO** will discuss and present to this subWG their preferred option moving forward (e.g. changes to A&I and/or symbology, no change, etc).
2. Other members are also invited to present their views for discussion at the next subWG meeting.

**Comment by RichardCoyles** [comment]
I'm seeking opinion with regard to the following Options and I'm currently discussing within the UKHO:

Option 1 - No Change to A&I - Dumping Ground shown below in S-101 - no alarms or indication in (W)ECDIS

![Capture1](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/108347351/24576fff-aea6-4122-8553-e23543fa5727)

Option 2 - No Change to A&I - Dumping Ground as above in S-101 - no alarms or indication in (W)ECDIS BUT also create a Caution Area (below), already included in A&I, with the same geometry as the Dumping Ground (Double Capture)

![Capture2](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/108347351/8cd2d6df-316f-4d23-a4a7-8f2bf070be3c)

Option 3 - Add Dumping Ground to Alarms and Indications in (W)ECDIS - No need to capture additional Cautionary Area. (Considering the need to reduce A&I in ECDIS, I would add that these A&I would be within specific Areas).

 comments welcome.

regards

Richard

**Comment by alvarosanuy** [comment]
> I'm seeking opinion with regard to the following Options and I'm currently discussing within the UKHO:

@RichardCoyles - Based on the latest encoding guidance in the DCEG I would say that HOs will have to manage these areas in different ways, based on their timely access (or not) to post dredging surveys of dumping grounds. If the HO knows charted data does not reflect latest post dumping depths they will need to opt from: downgrading the ZOC, amending the area to Unsurveyed area, double encoding a CTNARE, etc. Those countries able to release timely ENC updates based on latest surveys would do nothing.
In short, we are of the opinion that, allocating **DumpingGround** to A&I is not a good option. When there is evidence that charted depths do not reflect latest depths in the area, HOs should use alternative/additional encodings (as recommended by DCEG) to deliver the most appropriate depiction and alerts to mariners.

**Comment by alvarosanuy** [comment]
**Decisions made at Portrayal subWG meeting on 10/5/23**

Close this issue with NFA.
New specific proposals are to be submitted by opening a new GitHub issue.

**Comment by RichardCoyles** [comment]
Alvaro,
Your summary agrees in most part the views of technical leads at UKHO, there is no appetite for additional alarms, or  double capture with a cautionary area (only in exceptional cases) Most responses favour downgrading of the CATZOC. I agree dumping ground areas need to be managed on a case by case basis, there being sufficient guidance in the DCEG to cover all scenarios. We consider this issue closed. Regards Richard
